src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -27,5 +27,6 @@ do
     buildRootFSContainer="rootfs-$arch-$crossToolset"
     docker run --privileged --rm --name $buildRootFSContainer -e ROOTFS_DIR=/rootfs/$arch \
         -v $PWD/rootfs:/rootfs -v $scriptsVolume:/scripts \
-        $dockerCrossDepsTag /scripts/cross/build-rootfs.sh $arch $crossToolset $lldb --skipunmount
+        $dockerCrossDepsTag /scripts/cross/build-rootfs.sh $arch $crossToolset $lldb --skipunmount || \
+        exit 1
```

Comment:
exit on error

Reply:
@sdmaclea - would you mind adding a check to ensure $PWD/rootfs/$arch exists and is not empty after the docker run cmd?

Reply:
opened #25 to address the issue

Reply:
The two PRs should get merged together so that we can run CI against the full set of changes we believe is necessary.

Reply:
Done